presets/4.3/tune/whoop_ayyykayyy.txt

```diff
@@ -0,0 +1,130 @@
+#$ TITLE: Whoop race tune by AyyyKayyy
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: whoop, 65mm, 1S, race, acro, tune, AyyyKayyy, meteor, whooptopia, brushless
+#$ AUTHOR: AyyyKayyy - Anthony Knight
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/174/
+
+#$ DESCRIPTION: 65mm 1S brushless whoop acro tune for racing.
+#$ DESCRIPTION:
+#$ DESCRIPTION: Recommended gear this tune works the best with:
+#$ DESCRIPTION: 22000 - 25000kv motors, Meteor65 frame or simimlar, HQ tri-blades props.
+#$ DESCRIPTION: Works great with BBBLV3, BetaAIO 1s or HappyModel Diamond FC.
+#$ DESCRIPTION:
+#$ DESCRIPTION: No RPM filters requires, works with regular BLHeli_S. Filters checkbox can be unselected if you would like to keep your current filters.
+#$ DESCRIPTION:
+#$ DESCRIPTION: If you run a fast RC link like Redpine 666hz or ERLS, please consider also applying a corresponding RC_LINK on top of this preset.
+#$ DESCRIPTION:
+#$ DESCRIPTION: AyyyKayyy - Whooptopia winner 2019, 2020, and 2021. Whooptopia - biggest and the most intense whoop race in the USA.
+
+# ---------- TUNE DEFAULTS:
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+# ---------- END TUNE DEFAULTS
+
+#$ OPTION BEGIN (CHECKED): Filters
+    # ---------- FILTERS DEFAULTS:
+    #$ INCLUDE: presets/4.3/filters/defaults.txt
+    # ---------- END FILTERS DEFAULTS
+
+    # -- Gyro Dynamic Notches --
+    set dyn_notch_q = 250
+    set dyn_notch_min_hz = 90
+#$ OPTION END
+
+# -- PID values
+set p_pitch = 83
+set i_pitch = 162
+set d_pitch = 79
+set f_pitch = 171
+set p_roll = 76
+set i_roll = 153
+set d_roll = 72
+set f_roll = 162
+set p_yaw = 108
+set i_yaw = 162
+set f_yaw = 162
+set d_min_roll = 72
+set d_min_pitch = 79
+
+# -- PID Sliders
+set simplified_pids_mode = OFF
+
+# -- iTerm relax
+set iterm_relax_type = GYRO
+set iterm_relax_cutoff = 20
+
+# -- iTerm rotation
+set iterm_rotation = ON
+
+# -- TPA (default)
+set tpa_breakpoint = 1250
+
+# -- Antigravity
+set anti_gravity_gain = 0
```

Comment:
You sure you want  antigravity set to zero in 4.3, It works different to 4.2

Reply:
> You sure you want antigravity set to zero in 4.3, It works different to 4.2

let me ask lol

Reply:
> You sure you want antigravity set to zero in 4.3, It works different to 4.2

@sugaarK  i just flown this tune, its a real fire. it does not need AG :)